client/utils/scores.js

```diff
@@ -0,0 +1,8 @@
+export const computeTotalScore = (scores) => {
```

Comment:
Is the virtual property not already doing this ?

Reply:
Yes couldn’t get it on the trips so I just replaced it